etc/kayobe/ansible/wazuh-manager.yml

```diff
@@ -0,0 +1,70 @@
+---
+# Certificates generation
+- hosts: localhost
+  roles:
+    - role: "{{ playbook_dir }}/roles/wazuh-ansible/wazuh-ansible/roles/wazuh/wazuh-indexer"
+      perform_installation: false
+  become: no
+  tags:
+    - generate-certs
+# Single node
+- hosts: wazuh-master
+  become: yes
+  become_user: root
+  roles:
+    - role: "{{ playbook_dir }}/roles/wazuh-ansible/wazuh-ansible/roles/wazuh/wazuh-indexer"
+    - role: "{{ playbook_dir }}/roles/wazuh-ansible/wazuh-ansible/roles/wazuh/ansible-wazuh-manager"
+    - role: "{{ playbook_dir }}/roles/wazuh-ansible/wazuh-ansible/roles/wazuh/ansible-filebeat-oss"
+    - role: "{{ playbook_dir }}/roles/wazuh-ansible/wazuh-ansible/roles/wazuh/wazuh-dashboard"
+  post_tasks:
+    - name: Set http/s_proxy vars in ossec-init.conf for vulnerability detector
+      blockinfile:
+        path: "/var/ossec/etc/ossec.conf"
+        state: present
+        owner: root
+        group: ossec
+        block: |
+          HTTPS_PROXY={{ http_proxy_url }}
+          HTTP_PROXY={{ http_proxy_url }}
+        backup: yes
+      when: http_proxy_url is defined
+      notify:
+        - Restart wazuh
+
+    - name: Check if custom certificates exists
+      stat:
+        path: "{{ local_custom_certs_path }}"
+      register: custom_certificates_folder
+      delegate_to: localhost
+      become: no
+
+    - name: Copy the node & admin certificates to Wazuh dashboard
+      copy:
+        src: "{{ local_custom_certs_path }}/{{ item }}"
+        dest: /etc/wazuh-dashboard/certs/
+        owner: wazuh-dashboard
+        group: wazuh-dashboard
+        mode: 0400
+      with_items:
+        - "{{ indexer_node_name }}-key.pem"
+        - "{{ indexer_node_name }}.pem"
+      when: custom_certificates_folder.stat.exists
+      notify:
+        - Restart wazuh-dashboard
```

Comment:
Is there no way to do this on the first deploy? Could be something to follow up with